gizra.cabal

```diff
@@ -45,8 +44,13 @@ executable gizra
     , PartialTypeSignatures
     , StandaloneDeriving
     , DerivingVia
+
+executable gizra
+  import: settings
+  main-is: Main.hs
   build-depends: 
     ihp,
     base,
     wai,
-    text
+    text,
+    xss-sanitize
```

Comment:
Why is this added here?

Reply:
Using it here: https://github.com/klarkc/gizra/blob/1eb53effa3c64a0a30e53f33a1389a77274f5b72/Application/Helper/View.hs#L4-L9

Reply:
I mean, why on the cabal file and not on `default.nix`?

Reply:
It is here https://github.com/klarkc/gizra/blob/1eb53effa3c64a0a30e53f33a1389a77274f5b72/flake.nix#L53 (`default.nix` was moved to the flake).

AFAIK cabal file adds the dependency, nix provides the package to cabal consume, that's the reason why it's duplicated (in cabal and in nix).